test/test_ops.py

```diff
@@ -192,6 +228,48 @@ def wrapped_fn(*args, **kwargs):
 
             self.assertEqual(result, expected)
 
+    @ops(functorch_lagging_op_db + additional_op_db, allowed_dtypes=(torch.float,))
+    @skipOps('TestOperators', 'test_jvp', set({
+        # Apprently these support forward AD, but we get "Trying to use forward AD..."
+        xfail('var_mean'),
+        xfail('std_mean'),
```

Comment:
Do they? I don't see "result: " entries in derivatives.yaml

https://github.com/pytorch/pytorch/blob/3e3bf40b0a5acc68db40d0c2570207f2f35ae557/tools/autograd/derivatives.yaml#L1524-L1526

Reply:
```
import torch
import torch.autograd.forward_ad as fwAD

x = torch.randn(3)
y = torch.randn(3)

with fwAD.dual_level():
  x_dual = fwAD.make_dual(x, tx)
  torch.std_mean(x_dual)
```
gives me:
```
NotImplementedError: Trying to use forward AD with std_mean that does not support it.
```

Reply:
lmao the OpInfo has `supports_forward_ad=True`, but grad, gradgrad checks are all disabled